src/components/Select/Select.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css, cx } from 'react-emotion';
+
+import { textMega } from '../../styles/style-helpers';
+import IconInputWrapper from '../IconInputWrapper';
+import DownTriangleIcon from './down-triangle.svg';
+
+const baseSelectStyles = ({ theme }) => css`
+  label: select;
+  appearance: none;
+  background-color: ${theme.colors.white};
+  border-width: 1px;
+  border-style: solid;
+  border-color: ${theme.colors.n300};
+  border-radius: ${theme.borderRadius.mega};
+  box-shadow: inset 0 1px 2px 0 rgba(102, 113, 123, 0.21);
+  color: ${theme.colors.n900};
+  padding: ${theme.spacings.byte} ${theme.spacings.tera} ${theme.spacings.byte}
+    ${theme.spacings.kilo};
+  position: relative;
+  ${textMega({ theme })};
+  z-index: 30;
+
+  &:focus,
+  &:hover,
+  &:active {
+    outline: none;
+  }
+
+  &:focus {
+    border: 1px solid ${theme.colors.b500};
+  }
+
+  &::after {
+  }
+
+  &:-moz-focusring {
+    color: transparent;
+    text-shadow: 0 0 0 #000;
+  }
+`;
+
+const disabledSelectStyles = ({ disabled }) =>
+  disabled &&
+  css`
+    label: select--disabled;
+    opacity: 0.4;
+    pointer-events: none;
+  `;
+
+const baseIconStyles = ({ theme }) => css`
+  label: select__icon;
+  fill: ${theme.colors.n700};
+  display: block;
+  z-index: 40;
+  pointer-events: none;
+`;
+
+const SelectElement = styled('select', { label: 'SelectElement' })(
+  baseSelectStyles,
+  disabledSelectStyles
+);
+const Icon = styled(DownTriangleIcon, { label: 'SelectIcon' })(baseIconStyles);
+
+/**
+ * A native select component.
+ */
+const Select = ({
+  options,
+  value,
+  placeholder,
+  selector,
+  disabled,
+  ...props
+}) => (
+  <IconInputWrapper
+    {...{ selector }}
+    iconPosition="right"
```

Comment:
How about turning this position into a constant inside `IconInputWrapper`?

Then who uses the `IconInputWrapper` would know the options he has right always. Plus it avoids annoying bugs caused by misspellings (ex: "rigth" <img src="https://konlinejobs.com/wp-content/uploads/2017/02/funny-rage-face-1.jpg" width="20" />):
```js
<IconInputWrapper
  iconPosition={IconInputWrapper.RIGHT}
/>
<IconInputWrapper
  iconPosition={IconInputWrapper.LEFT}
/>
...
```

Reply:
I like this idea and already used it in the `Button` component. 👍